openfisca_france_local/convert_benefit_conditions_to_parameters.py

```diff
@@ -105,10 +105,34 @@ def conditions_to_node_data(conditions: 'list[dict]') -> dict:
 
 def profils_to_node_data(profils: 'list[dict]'):
     def create_profils_field(data: dict, profil: dict):
-        data['profils'].update({profil['type']: {}})
+        if profil['type'] not in data['profils']:
+            data['profils'].update({profil['type']: {}})
 
     def add_profil_with_conditions(data: dict, profil: dict):
-        data['profils'][profil['type']].update(conditions_to_node_data(profil['conditions']))
+        def condition_already_exists_in_node(profil_condition, conditions_in_node_data) -> bool:
+            for condition_type in profil_condition.keys():
+                if condition_type in conditions_in_node_data:
+                    conditions_with_operator_fields = ['age', 'quotient_familial', 'situation_handicap']
```

Comment:
cette définition on pourrait la remonté avant le for, ça rendrait un peu mieux j'ai l'impression, t'en pense quoi ?